README.md

```diff
@@ -6,23 +6,28 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/wix7wml3v55670kw?svg=true)](https://ci.appveyor.com/project/TooTallNate/node-speaker)
 
 A Writable stream instance that accepts [PCM audio][pcm] data and outputs it
-to the speakers. The output is backed by `mpg123`'s audio output modules, which
-in turn use any number of audio backends commonly found on Operating Systems
-these days.
+to the speakers. The output is backed by [libao][ao], which supports
+a ton of audio backends.
+
 
 ## Installation
 
-Simply compile and install `node-speaker` using `npm`:
+You need `libao` installed on your system before installig `node-speaker`.
```

Comment:
This is possibly a bit of a drawback, I kind of liked the approach of bundling the source to the lib.

How do we feel about just adding the libao source code to this repo?

Reply:
### A bundle of joy

Hum, well, I always thought bundling was a Bad Idea, but I see your point about ease of use. So, I've done some thinking and reviewing my stand on bundling dependencies.

#### Here's what I've come up with

The whole point of switching to libao is to avoid bundled sources. The tl;dr version of why is because once you bundle the sources, you kind of own the dependency and get all the maintenance burden (which we already have, as the issues show). On the other side, proper external dependencies allow us to report issues in the right place and let the proper owners deal with it.

#### Conclusion

It is comforting for users, but I'm not willing the pay the price.

#### References:

* https://lukeplant.me.uk/blog/posts/bundling-dependencies/
* https://wiki.gentoo.org/wiki/Why_not_bundle_dependencies